### PR TITLE
Select collection screen - multi select, repository field, repo filter

### PR DESCRIPTION
**Pull request by jerabekjiri**
Issue: [AAH-2274](https://issues.redhat.com/browse/AAH-2274)

- multi select and repository column:
Screenshot from 2023-04-13 14-19-51

Screenshot from 2023-04-13 14-18-07

### CODE REVIEW

File: src/actions/ansible-repository-collection-version-add.tsx [hunk 1]
```diff
@@ -8,29 +8,43 @@ import {
...
       );
-      setState((ms) => ({ ...ms, addCollectionVersionModal: null }));
-      query({});
     })
     .catch(
       handleHttpError(
-        t`Failed to add ${namespace}.${name} v${version} to repository "${repositoryName}".`,
+        t`Failed to add collection${
+          collections.length === 1 ? '' : 's'
+        } to repository "${repositoryName}".`,
```
Comment (himdel) [inline src/actions/ansible-repository-collection-version-add.tsx:47]: this is problematic when translating to languages which do not use -s for plurals

I guess the proper way with plurals would be...

`import { plural } from '@lingui/macro';`

```suggestion
    plural(collections.length, {
      one: 'Failed to add collection to repository "{repositoryName}".',
      other: 'Failed to add collections to repository "{repositoryName}".',
    }).replace('{repositoryName}', repositoryName)
```

(unless I'm missing a nice way to add interpolation to plurals)

**Comment by himdel** [comment]
The rest looks good :+1:

File: src/actions/ansible-repository-collection-version-add.tsx (hunk 1 above)
Comment (jerabekjiri) [reply src/actions/ansible-repository-collection-version-add.tsx]: Forgot about this, this is much better better, thanks :+1:

Comment (jerabekjiri) [reply src/actions/ansible-repository-collection-version-add.tsx]: seems like `.replace('{repositoryName}', repositoryName),` doesn't replace variable in the string, but it works like this  
```
 plural(collections.length, {
    one: `Failed to add collection to repository "${repositoryName}".`,
    other: `Failed to add collections to repository "${repositoryName}".`,
})
```

**Pull request merged**